app/features/map-planner/components/Planner.tsx

```diff
@@ -34,7 +29,7 @@ export default function Planner() {
   const { i18n } = useTranslation();
   const plannerBgParams = usePlannerBg();
 
-  const [app, setApp] = React.useState<TldrawApp | null>(null);
+  const [app, setApp] = React.useState<TldrawApp>(new TldrawApp());
```

Comment:
I prefer defining an empty default value for our Tldrawapp object here, just so we won't have to deal with `null` related checks if we plan to use the `app` state more often.